src/orders/components/OrderFulfillment/TableLine.tsx

```diff
@@ -0,0 +1,124 @@
+import { makeStyles, TableCell, TableRow } from "@material-ui/core";
+import Money from "@saleor/components/Money";
+import Skeleton from "@saleor/components/Skeleton";
+import TableCellAvatar, {
+  AVATAR_MARGIN
+} from "@saleor/components/TableCellAvatar";
+import { maybe } from "@saleor/misc";
+import {
+  OrderDetails_order_fulfillments_lines,
+  OrderDetails_order_lines
+} from "@saleor/orders/types/OrderDetails";
+import React from "react";
+
+const useStyles = makeStyles(
+  theme => ({
+    clickableRow: {
+      cursor: "pointer"
+    },
+    colName: {
+      width: "auto"
+    },
+    colNameLabel: {
+      marginLeft: AVATAR_MARGIN
+    },
+    colPrice: {
+      textAlign: "right",
+      width: 120
+    },
+    colQuantity: {
+      textAlign: "center",
+      width: 120
+    },
+    colSku: {
+      textAlign: "right",
+      textOverflow: "ellipsis",
+      width: 120
+    },
+    colTotal: {
+      textAlign: "right",
+      width: 120
+    },
+    infoLabel: {
+      display: "inline-block"
+    },
+    infoLabelWithMargin: {
+      marginBottom: theme.spacing()
+    },
+    infoRow: {
+      padding: theme.spacing(2, 3)
+    },
+    orderNumber: {
+      display: "inline",
+      marginLeft: theme.spacing(1)
+    },
+    statusBar: {
+      paddingTop: 0
+    },
+    table: {
+      tableLayout: "fixed"
+    }
+  }),
+  { name: "TableLine" }
+);
+
+interface TableLineProps {
+  line: OrderDetails_order_fulfillments_lines | OrderDetails_order_lines;
+  isOrderLine?: boolean;
+}
+
+const TableLine: React.FC<TableLineProps> = ({
```

Comment:
Maybe FullfillmentTableLine would be more descriptive name? Helpful especially when using auto-import for code completion.

Reply:
Yeeees...and no. It's a table line for both fulfillment and order line, and since in most of the code naming something 'fulfillment' maps directly to actual fulfillment, would rather not call it that cause then it would be misleading. But will move the table elements to some suitable directory so the whole dir will be named correctly

Reply:
Then OrderTableRow would be good, since it cover different states (fullfillment, etc). My gripe is that we have Customer tables, attribute tables, etc

Reply:
Changed to *even better ones*